### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Aayush-Ovhal** [neverhood311/Stop-motion-OBJ#152] [General]
Man, thanks a lot for making such an amazing add-on! It really solved the issue I had...I can't thank you enough for how much you have helped me.

**Comment by neverhood311** [comment]
Thank you for the heartwarming message!

I'm glad that Stop Motion OBJ has made your work a little easier.

**Reply by RPLgrime** [reply]
Wow, I think this is going to be ridiculously useful to me! I'm going to test it out and check back in, but I'm so excited right now!

**Reply by neverhood311** [reply]
Can't wait to hear how it goes
